apim-operator/pkg/controller/api/api_controller.go

```diff
@@ -224,6 +224,18 @@ func (r *ReconcileAPI) Reconcile(request reconcile.Request) (reconcile.Result, e
 		}
 	}
 
+	if security.Spec.Type == "JWT" {
+
+		certificateAlias = security.Name + "alias"
+
```

Comment:
Shall we remove the extra lines 228 & 230?

Reply:
This is already fixed in latest code